pieces.sublime-settings

```diff
@@ -0,0 +1,15 @@
+{
+	// "host": "http:localhost/1000",
+	"model":"GPT-3.5-turbo Chat Model" 
```

Comment:
You can change the model from here and the host too if you did not add any host it will use the default ones